templates/vertical-full-page-map/markup/searchthisareabutton.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="Answers-searchThisAreaWrapper">
   <button id="Answers-searchThisAreaButton" type="button" class="Answers-searchThisArea js-searchThisAreaButton">
-    Search This Area
+    {{ translate phrase="Search This Area" context="Button to conduct a search in the current map area" }}
```

Comment:
nit context suggestion: 'A button which conducts a search in the current map area'